docs/se/grpc/server.adoc

```diff
@@ -648,7 +641,7 @@ GrpcRouting routing = GrpcRouting.builder()
                 .nameFunction((svc, method, metric) -> "grpc." + service.name() + '.' + method) // <1>
 ----
 <1> the `NamingFunction` is just a lambda that returns the concatenated service name and method name
-with the prefix `grpc.` So for a service "Foo", method "bar" the above example would produce a name "grpc.Foo.bar".
+with the prefix ``grpc.``. So for a service "Foo" and method "bar", the above example would produce a name "grpc.Foo.bar".
 
 === Security
 To enable Server Security, refer to earlier section about <<security_maven_coordinartes, Security maven coordinates>> for guidance on what dependency to add in the project's pom.xml.
```

Comment:
647: "Server Security" -> "server security" and "to earlier" -> "to the earlier"
651: "with gRPC" -> "with the gRPC"
653: "Create security instance" -> "Create the security instance" and "with server" -> "with the server." (note added period at end)
654: "Protect" -> "Protect the" and "of server" -> "of the server" and add ending period